.drone/drone.yml

```diff
@@ -0,0 +1,142 @@
+---
+kind: pipeline
+name: Lint
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: lint
+  image: golangci/golangci-lint:v1.37.1
+  commands:
+  - apt-get update -y && apt-get install -y libsystemd-dev
+  - make lint
+---
+kind: pipeline
+type: docker
+name: Dist
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: distribute
+  image: rfratto/seego
+  commands:
+  - apt-get update &&  apt-get install -y rubygems rpm nsis
+  - gem install --no-document fpm
+  - mkdir -p /usr/local/go/bin
+  - wget -q https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+  - tar -C /usr/local -xzf go1.16.5.linux-amd64.tar.gz
+  - rm go1.16.5.linux-amd64.tar.gz
```

Comment:
nit: since this isn't a Dockerfile we probably don't have to remove the tarball, though it doesn't really matter to me if we leave this in or not.

Reply:
Its really annoying with drone =) If you run multiple drone instances it created a ton of .gz files that sit around.

Reply:
Ah, gotcha 👍